src/form-layout-grid.scss

```diff
@@ -0,0 +1,165 @@
+@import "./mixins";
+@import "./layout-grid.scss";
+
+$block: #{$fd-namespace}-form-layout-grid;
+$blockCol: #{$fd-namespace}-col;
+$blockContainer: #{$fd-namespace}-container;
+$blockRow: #{$fd-namespace}-row;
+$cols: 11;
+
+@media (min-width: 0) {
+  .#{$blockContainer}.#{$block}-container {
+    padding: 1rem;
+    margin: 0 -0.25rem;
+
+    & > .fd-row {
+      & > :first-child & :not(.fd-col--wrap) {
+        display: flex;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+    }
+
+    .#{$blockRow},
+    .#{$blockRow}+.#{$blockRow} {
+      margin-top: 0;
+    }
+
+    .#{$blockRow} {
+      align-items: center;
+      margin-top: 0;
+      margin-bottom: 0.625rem;
+
+      .#{$blockCol} {
+        padding: 0 0.25rem;
+        text-align: left;
+        justify-content: flex-start;
+
+        .fd-select__control {
+          margin: 0.25rem 0;
+        }
+
+        .fd-form-label {
+          margin-right: 0;
+          padding-bottom: 0.125rem;
+        }
+
+        @for $n from 0 through $cols {
+          &--#{$n} {
+            text-align: right;
+            justify-content: flex-end;
+            margin-bottom: 0;
+
+            .fd-form-label {
+              padding-bottom: 0;
+            }
+          }
+        }
+
+        &--12 {
+          text-align: left;
+          justify-content: flex-start;
+        }
+
+        .#{$blockRow} {
+          margin: 0 -0.25rem;
+        }
+      }
+    }
+
+    :last-child {
+      * > {
+        margin-bottom: 0;
+      }
+    }
+  }
+}
+
+@media (min-width: 600px) {
+  .#{$blockContainer}.#{$block}-container {
+    .#{$blockRow} {
+      margin-bottom: 0;
+
+      .#{$blockCol} {
+        .fd-form-label {
+          padding-bottom: 0;
+        }
+
+        @for $n from 0 through $cols {
+          &--#{$n}--m {
+            text-align: right;
+            justify-content: flex-end;
```

Comment:
You are using this block of code rather often. I'd put it in a mixin like 
```
@mixin something($align, $justify) {
    text-align: $align;
    justify-content: $justify;
}
```

and then use it like this:
```
@include @something('right', 'flex-end') 
```